docs/graph-algorithms/bipartite-check.mdx

```diff
@@ -0,0 +1,245 @@
+---
+title: Bipartite Check
+slug: /graphs-algorithms/bipartite-check
+---
+
+## Introduction
+
+A bipartite graph is a graph whose vertices can be divided into two disjoint sets so that every edge connects two vertices from different sets (i.e. there are no edges which connect vertices from the same set). These sets are usually called sides.
+
+A graph is bipartite if and only if it is two-colorable. Equivalently, a bipartite graph is a graph that does not contain any *odd-length cycles*.
+
+In the Bipartite check problem, we are given an undirected graph. We need to determine whether the given graph is a Bipartite.
+
+[![Screenshot-82.png](https://i.postimg.cc/W4LJJ80T/Screenshot-82.png)](https://postimg.cc/QFmM2p6y)
```

Comment:
Have you made this image?

Reply:
Yes. I used [this](https://postimg.cc/) to get the url.